apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/task/DefaultTaskDispatcher.java

```diff
@@ -291,10 +291,10 @@ private void runAllAndWait(final Collection<Task> tasks) {
                 tasks.forEach(task -> {
                     onStartExecutor.submit(() -> {
                         try {
-                            log.trace("Current thread={}", Thread.currentThread().getName());
+                            log.debug("Task = {}, Current thread={}", task.getDelay(), Thread.currentThread().getName());
```

Comment:
Do you probably mean using the getName() method rather getDelay()?

Reply:
yes, you are right, fixed to getName()